src/AlignLoads.cpp

```diff
@@ -66,30 +63,16 @@ class AlignLoads : public IRMutator2 {
             // non-constant strides.
             return IRMutator2::visit(op);
         }
-        int lanes = ramp->lanes;
-        int native_lanes = required_alignment / op->type.bytes();
 
-        if (!(*const_stride == 1 || *const_stride == 2 || *const_stride == 3)) {
```

Comment:
I think this logic has been lost, I think we'll rewrite loads even with huge constant strides whereas before we did not.

Reply:
Nice catch, thanks.